client/my-sites/checkout/checkout-thank-you/akismet-checkout-thank-you.tsx

```diff
@@ -87,4 +97,48 @@ const AkismetCheckoutThankYou: FunctionComponent< AkismetCheckoutThankYouProps >
 	);
 };
 
+function ThankYouAPIKeyClipboard() {
+	const translate = useTranslate();
+	const [ isCopied, setCopied ] = useState( false );
+	const { data, isError, isLoading } = useAkismetKeyQuery();
+
+	useEffect( () => {
+		if ( isCopied ) {
+			const confirmationTimeout = setTimeout( () => setCopied( false ), 4000 );
+			return () => clearTimeout( confirmationTimeout );
+		}
+	}, [ isCopied ] );
+
+	const showConfirmation = () => {
+		setCopied( true );
+	};
+
+	if ( isError ) {
+		return null;
+	}
+
+	const akismetApiKey = data ?? '';
+	const keyInputSize = akismetApiKey ? akismetApiKey.length + 5 : 0;
+
+	return (
+		<div
+			className={ 'akismet-checkout-thank-you__key-clipboard' + ( isLoading ? ' loading' : '' ) }
```

Comment:
For multiple classnames I like importing
```tsx
import classNames from 'classnames';
```
So they can be added in a comma separated list. Not required by any stretch though

Reply:
TIL about `classnames`!